### PR TITLE
Add example for ecs standalone task

### PR DESCRIPTION
**Pull request by TonkyH**
**What this PR does / why we need it**:
Add example for ecs standalone task and docs

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md [hunk 1]
```diff
@@ -6,14 +6,47 @@ description: >
...
 
 If you're not familiar with ECS, you can get examples for those files from [here](../../../../examples/#ecs-applications).
 
 ## Quick sync
 
 By default, when the [pipeline](../../../configuration-reference/#ecs-application) was not specified, PipeCD triggers a quick sync deployment for the merged pull request.
 Quick sync for an ECS deployment will roll out the new version and switch all traffic to it immediately.
+> In case of standalone task, only Quick sync is supported.
+
+Here is an example of running standalone task:
```
Comment (khanhtc1202) [inline docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md:28]: How about using tab switchable to show configuration example for QuickSync in both standalone and application cases like this? 🤔 
https://pipecd.dev/docs/installation/install-piped/installing-on-kubernetes/#:~:text=syncInterval%3A%201m-,Installing%20by%20using,-Helm%20(3.8.0%20or

Comment (TonkyH) [reply docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md:28]: That sounds great!🚀
I will try to use tab switchable.

File: docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md [hunk 2]
```diff
@@ -6,14 +6,47 @@ description: >
...
+To deploy an Amazon ECS application, `TaskDefinition` configuration file must be located in the application directory. This file contains all configuration for [ECS TaskDefinition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html) object and will be used by Piped agent while deploy your application/service to ECS cluster.
+
+> To deploy your application to run continually or to place it behind a load balancer, You need to create [ECS Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html). `Service` configuration file also must be located in the application directory. This file contains all configuration for [ECS Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html) object.
+
+There are two main ways to deploy an Amazon ECS application.
+- Your application is a one-time or periodic batch job.
+  - it's a standalone task.
+  - you need to prepare `TaskDefinition`
+- Your application is deployed to run continuously or behind a load balancer.
+  - you need to prepare `TaskDefinition` and `Service`
```
Comment (khanhtc1202) [inline docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md:18]:
```suggestion
There are two main ways to deploy an Amazon ECS application.
- Your application is a one-time or periodic batch job.
  - it's a standalone task.
  - you need to prepare `TaskDefinition`
- Your application is deployed to run continuously or behind a load balancer.
  - you need to prepare `TaskDefinition` and `Service`

To deploy an Amazon ECS application, the `TaskDefinition` configuration file must be located in the application directory. This file contains all configuration for [ECS TaskDefinition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html) object and will be used by Piped agent while deploying your application/service to the ECS cluster.

To deploy your application to run continuously or to place it behind a load balancer, You need to create [ECS Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html). The `Service` configuration file also must be located in the application directory. This file contains all configurations for [ECS Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html) object.
```
I think we can revert those blocks like this to make it easy to follow

Comment (TonkyH) [reply docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md:18]: Yes, it's easier to understand when blocks are organized🙇
I'll rewrite it🚀

File: docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md [hunk 3]
```diff
@@ -6,14 +6,69 @@ description: >
...
 
+To deploy an Amazon ECS application, the `TaskDefinition` configuration file must be located in the application directory. This file contains all configuration for [ECS TaskDefinition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html) object and will be used by Piped agent while deploying your application/service to the ECS cluster.
+
+To deploy your application to run continuously or to place it behind a load balancer, You need to create [ECS Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html). The `Service` configuration file also must be located in the application directory. This file contains all configurations for [ECS Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html) object.
```
Comment (khanhtc1202) [inline docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md:18]:
```suggestion
To deploy your application to run continuously or to place it behind a load balancer, You need to create [ECS Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html). The `Service` configuration file also must be located in the application directory. This file contains all configurations for [ECS Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html) object.

```
nits

File: docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md [hunk 4]
```diff
@@ -46,7 +101,6 @@ kind: ECSApp
 spec:
   input:
     # Path to Service configuration file in Yaml/JSON format.
-    # Default is `service.json`
```
Comment (khanhtc1202) [inline docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md:49]: Could you also check other examples and remove this? I think this note exists in other places too 👀

Comment (TonkyH) [reply docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md:49]: It was in other versions, but not in docs-dev!

スクリーンショット 2023-01-06 14 33 22

**Review by khanhtc1202 — APPROVED** [review]
Nice 👍

**Pull request merged**